### PR TITLE
release note: V3.3.0: STM32 and shields update

### PR DESCRIPTION
**Pull request by erwango**

### CODE REVIEW

**Comment by stephanosio** [comment]
@erwango There is a documentation build failure:
```
/home/runner/work/zephyr/zephyr/doc/_build/src/releases/release-notes-3.3.rst:314:undefined label: 'counter'
```

File: doc/releases/release-notes-3.3.rst [hunk 1]
```diff
@@ -355,13 +393,21 @@ Drivers and Sensors
 
 * ADC
 
+  * STM32: Now Supports sequencing multiple channels into a single read.
+
+* Battery-backed RAM
+
+  * STM32: Add driver to enable support for backup registers from RTC.
```
Comment (stephanosio) [inline doc/releases/release-notes-3.3.rst:400]: Please use the past tense to be consistent with the rest.

```suggestion
  * STM32: Added driver to enable support for backup registers from RTC.
```

File: doc/releases/release-notes-3.3.rst [hunk 2]
```diff
@@ -882,6 +967,12 @@ Libraries / Subsystems
 HALs
 ****
 
+* STM32
+
+  * stm32cube: update stm32h7 to cube version V1.11.0.
```
Comment (stephanosio) [inline doc/releases/release-notes-3.3.rst:972]:
```suggestion
  * stm32cube: updated stm32h7 to cube version V1.11.0.
```

**Review by stephanosio — CHANGES_REQUESTED** [review]

File: doc/releases/release-notes-3.3.rst (hunk 1 above)
Comment (erwango) [reply doc/releases/release-notes-3.3.rst]: Right, txs. Fixed

File: doc/releases/release-notes-3.3.rst [hunk 3]
```diff
@@ -132,6 +136,9 @@ Removed APIs in this release
 
 * Removed deprecated ``DT_CHOSEN_*_LABEL`` helper macros.
 
+* Remove deprecated property ``enable-pin-remap`` from  :dtcompatible: `st,stm32-usb`:.
```
Comment (stephanosio) [inline doc/releases/release-notes-3.3.rst:139]:
```suggestion
* Removed deprecated property ``enable-pin-remap`` from  :dtcompatible: `st,stm32-usb`:.
```

**Review by stephanosio — COMMENTED** [review]
one more :)

**Review by stephanosio — DISMISSED** [review]

**Review by nashif — DISMISSED** [review]

**Comment by stephanosio** [comment]
@erwango This needs a rebase.

**Review by laurenmurphyx64 — DISMISSED** [review]

**Review by carlescufi — DISMISSED** [review]

**Comment by stephanosio** [comment]
@erwango Need another rebase :(

**Review by stephanosio — APPROVED** [review]

**Review by FRASTM — APPROVED** [review]

**Pull request merged**